src/app/volunteerRequest.ts

```diff
@@ -15,6 +17,18 @@ export const getVolunteerRequestsOfProgram = async (
   return volunteerRequestRepository.requestsOfProgram(programId, institutionId, startDate);
 };
 
+export const createVolunteerRequest = async (volunteerRequest: VolunteerRequest): Promise<VolunteerRequest> => {
+  return volunteerRequestRepository.save(validateSchema(volunteerRequestSchema, volunteerRequest));
```

Comment:
missing checks for creation of the volunteer request, for example, if the user is from institution id 1, then the volunteer request must have also institution id 1. I'm sure there are more validations to make besides that.